views/js/controller/admin/index.js

```diff
@@ -269,20 +269,16 @@ define([
 
                 $submitter.addClass('disabled');
 
-                $.ajax({
+                // submit form with tokenised request
+                request({
```

Comment:
Since the promise isn't returned you should keep calling `catch`  and handling the error

Reply:
Thanks, fixed now.